pkgs/tools/misc/diffoscope/default.nix

```
@@ -54,10 +55,20 @@ python3Packages.buildPythonApplication rec {
       python-magic progressbar33 pypdf2 tlsh
     ])
     ++ lib.optionals stdenv.isLinux [ python3Packages.pyxattr python3Packages.rpm acl cdrkit dtc ]
-    ++ lib.optionals enableBloat ([
-      abootimg apksigcopier apksigner apktool cbfstool colord enjarify ffmpeg fpc ghc ghostscriptX giflib gnupg gnumeric
-      hdf5 imagemagick libcaca llvm jdk mono ocaml odt2txt oggvideotools openssh pdftk poppler_utils procyon qemu R tcpdump ubootTools wabt radare2 xmlbeans
-    ] ++ (with python3Packages; [ androguard binwalk guestfs h5py pdfminer-six ]));
+    ++ lib.optionals enableBloat (
+      [
+        apksigcopier apksigner enjarify ffmpeg fpc ghc ghostscriptX giflib gnupg pdftk
+        hdf5 imagemagick libcaca llvm jdk mono ocaml odt2txt openssh
+        poppler_utils procyon qemu R tcpdump wabt radare2 xmlbeans
+      ]
+      ++ (with python3Packages; [ androguard binwalk h5py pdfminer-six ])
+      # oggvideotools is broken on Darwin, please put it back when it will be fixed?
+      ++ lib.optionals stdenv.isLinux [ abootimg cbfstool colord ubootTools python3Packages.guestfs oggvideotools ]
+      # This doesn't work on aarch64-darwin
+      ++ lib.optionals (stdenv.isx86_64 || stdenv.isLinux) [ gnumeric ]
```

Unless it does something specifically to only work on aarch64 on Linux, this doesn't mean it should be disabled for all non-Linux OSes on aarch64:

```suggestion
      ++ lib.optionals (stdenv.hostPlatform.system != "aarch64-darwin") [ gnumeric ]
```

---

Right

---

Done